apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/oms/OmstillingsstoenadInnvilgelse.kt

```diff
@@ -64,7 +68,15 @@ data class OmstillingsstoenadInnvilgelse(
                     )
                 }
 
-            val avdoed = avdoede.single()
+            val tomAvdoed =
```

Comment:
Litt skeptisk til denne. Er det noen grunn til at du heller velger et mocket objekt her, enn å sette avdød som optional? Fra brevsiden så tenker jeg heller at man tenker seg om to ganger før man bruker avdød dersom den er optional, fremfor at den tilsynelatende er der men med fake info, noe man ikke nødvendigvis ser før man feks bruker doedsdato i et brev. Tanker?